### PR TITLE
installer: add adl and adl-s links

### PR DESCRIPTION
**Pull request by marc-hb**
See (latest) commit messages

### CODE REVIEW

File: installer/tests/staging_sof_ref.txt [hunk 1]
```diff
@@ -1,5 +1,7 @@
 .
 ├── community
+│   ├── sof-adl.ri -> sof-tgl.ri
+│   ├── sof-adl-s.ri -> sof-tgl.ri
```
Comment (keyonjie) [inline installer/tests/staging_sof_ref.txt:4]: sof-tgl-h.ri

File: installer/tests/staging_sof_ref.txt [hunk 2]
```diff
@@ -11,9 +13,14 @@
 │   ├── sof-tgl-h.ri
 │   └── sof-tgl.ri
 ├── intel-signed
+│   ├── sof-adl.ri -> sof-tgl.ri
 │   ├── sof-cfl.ri -> sof-cnl.ri
 │   ├── sof-cml.ri -> sof-cnl.ri
 │   └── sof-glk.ri -> sof-apl.ri
+├── sof-adl.ldc -> sof-tgl.ldc
+├── sof-adl.ri -> intel-signed/sof-adl.ri
+├── sof-adl-s.ldc -> sof-tgl.ldc
```
Comment (keyonjie) [inline installer/tests/staging_sof_ref.txt:22]: sof-tgl-h.ldc

**Comment by marc-hb** [comment]
The Zephyr build failure is due to the recent SDK requirement bump (0.13.1), unrelated  work in progress.

Checkpatch  is lost in the weeds.

**Comment by marc-hb** [comment]
The `/lib/firmware`-like output modified by this PR is the input to https://github.com/thesofproject/sof-bin/blob/main/HOWTO-new-release.md#release

File: installer/GNUmakefile [hunk 3]
```diff
@@ -22,7 +22,7 @@ SIGNED_list  ?= apl cnl icl jsl tgl tgl-h
```
Comment (kv2019i) [inline installer/GNUmakefile:22]: Commit msg: "Based on Mengdong's guidance". Proposal:

ADL and ADL-S binaries share FW build configuration with TGL and TGL-H.

**Review by kv2019i — CHANGES_REQUESTED** [review]
Thanks @marc-hb . Patch content looks, but I'd reword the second commit message to be a bit more formal. The HW details have been described earlier in commit "config: intel: use PCH name for tigerlake"

File: installer/GNUmakefile (hunk 3 above)
Comment (lgirdwood) [reply installer/GNUmakefile:22]: @mengdonglin

Comment (marc-hb) [reply installer/GNUmakefile:22]: Done

**Review by kv2019i — APPROVED** [review]
Perfect, thanks @marc-hb !

**Comment by lgirdwood** [comment]
This PR introduces installer updates and has no impact on CI runtime tests.
